qds_sdk/engine.py

```diff
@@ -71,7 +75,10 @@ def set_engine_config(self,
 
         '''
 
-        self.set_hadoop_settings(custom_hadoop_config, use_qubole_placement_policy, is_ha, fairscheduler_config_xml, default_pool, enable_rubix)
+        self.set_hadoop_settings(custom_hadoop_config, use_qubole_placement_policy,
+                                 is_ha, fairscheduler_config_xml,
+                                 default_pool, enable_rubix)
+        self.set_hive_settings(hive_version)
```

Comment:
Verify the value of hive_version column in hive_cluster_configs with and without this change.

Reply:
![image](https://user-images.githubusercontent.com/57746175/77913454-e8e01980-72b1-11ea-9494-2d8d144e2514.png)

@tarrygo above is the screenshot from DB with the updated sdk changes and hive version being updated with 3.1